pandas/core/arrays/arrow/array.py

```diff
@@ -48,16 +49,27 @@
 
 class ArrowExtensionArray(ExtensionArray):
     """
-    Base class for ExtensionArray backed by Arrow array.
+    Base class for ExtensionArray backed by Arrow ChunkedArray.
     """
 
     _data: pa.ChunkedArray
 
-    def __init__(self, values: pa.ChunkedArray) -> None:
-        self._data = values
+    def __init__(self, values: pa.Array | pa.ChunkedArray) -> None:
+        if pa_version_under1p01:
```

Comment:
we already have this as a min pyarrow version

Reply:
we should bump our min pyarrow anyhow

Reply:
Any suggestion what to bump our min pyarrow version too?

e.g. pyarrow 4 has been out for about 1 year cc @jorisvandenbossche

Reply:
kk can do this in another issue / pR

Reply:
AFAIK we already bumped the minimum version to 1.0, so I think this can just be removed? (see https://github.com/pandas-dev/pandas/pull/44064)

Reply:
This is defined as `pa_version_under1p01 = Version(pa.__version__) < Version("1.0.1")`, so this functions as our min version check I think

Reply:
Ah, of course, yes, here it is just to raise the informative error message (was misreading Jeff's question). So it's not a minimum version check in this case, but rather a check if pyarrow is installed or not (if pyarrow is not installed, we set `pa_version_under1p01` to True)